src/sign.builds

```diff
@@ -19,7 +19,7 @@
 
   <!-- apply the default signing certificates (defined in sign.targets) -->
   <ItemDefinitionGroup>
-    <FilesToSign>
+    <FilesToSign Condition="'$(BuildArch)' != 'arm64'">
```

Comment:
This is the item definition group, and while having this condition here won't hurt it isn't strictly necessary and won't give you the results you want.  You need to add this condition to the item groups where files to sign are included.

```
  <ItemGroup Condition="'$(BuildArch)' != 'arm64'">
    <FilesToSign Include="$(BinDir)*.dll" Exclude="$(BinDir)*.ni.dll" />
    <FilesToSign Include="$(BinDir)*.exe" />
  </ItemGroup>
```